### DISCUSSION TITLE
How to obtain point cloud data from the O3D303 camera and display it in the PCL viewer

### DISCUSSION THREAD

**Discussion by 520liangqiujuan** [ifm/ifm3d#457] [Q&A]
I want to know how to acquire point cloud data from the camera and display it in the PCL viewer,my ifm3d version is v1.5.3.

**Comment by lola-masson** [comment]
Hi @520liangqiujuan,

For the O3D303, you can use the examples available with the API documentation: https://api.ifm3d.com/stable/examples/o3r/getting_data/getting_data.html

In the examples, replace the `O3R` objects with the `O3D`, and when instantiating the `FrameGrabber`, leave the second argument (`pcic_port`) blank, and you should be good to go.
